lib/services/product_service.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+
+import '../models/product_model.dart';
+
+class ProductServices {
+  Future<List<Product>> getProducts() async {
+    List<Product> product = [];
```

Comment:
when you declare a **List** variable, it's preferable to naming it as plural name. eg : **products**